Utilities/StrFmt.cpp

```diff
@@ -286,6 +286,15 @@ struct fmt::cfmt_src
 
 		return 0;
 	}
+
+	static constexpr std::size_t size_char  = 1;
+	static constexpr std::size_t size_short = 2;
+	static constexpr std::size_t size_int   = 0;
```

Comment:
Why `0` (and why the duplication here and in `sys_libc_.cpp`? with same names but different values).